spec/factories/relays.rb

```diff
@@ -4,7 +4,7 @@
   factory :relay do
     type { "RelaySynaccessRevA" }
     ip { "192.168.1.1" }
-    sequence(:port) { |p| p }
+    sequence(:outlet) { |o| o }
```

Comment:
We didn't have a max value validation on this field before, so we didn't see any problems, but now if we create more than 16 of these from this factory, they'll start becoming invalid (I think that's what the majority of current test failures are). Maybe we could add a modulo in there: `sequence(:outlet) { |o| o % PowerRelay::MAXIMIUM_OUTLETS }`